from chess.piece.exceptions import PieceValidationError


class QueenValidationError(PieceValidationError):
    pass

